scripts/code_size_compare.py

```diff
@@ -163,6 +163,12 @@ def detect_arch() -> str:
 CRYPTO_CONFIG_H = 'include/psa/crypto_config.h'
 BACKUP_SUFFIX = '.code_size.bak'
 
+CONFIG_SCRIPT = './scripts/config.py'
+CONFIG_UNSET = 'unset'
+CONFIG_MACROS = [
```

Comment:
It's a bit non-obvious that `CONFIG_MACROS` would all be unset, rather than set. I suggest maybe something like:
```
CONFIG_MACROS = {
    "SET": [],
    "UNSET": [ 'MBEDTLS_PSA_CRYPTO_SPM' ]
}
```